lib/absinthe/type/built_ins/introspection.ex

```diff
@@ -84,8 +84,10 @@ defmodule Absinthe.Type.BuiltIns.Introspection do
       :field,
       :fragment_definition,
       :fragment_spread,
-      :inline_fragment
+      :inline_fragment,
       # TODO: Schema definitions to support Schema input
+      :object,
```

Comment:
Had to add these otherwise a SDL defined schema failed an introspection query

cc @bruce

Reply:
I think this _and_ the definition for the `Absinthe.Type.Directive.location()` type need to be updated to match the full list of directive locations, which are the possible return values from `Absinthe.Blueprint.Directive.placement/1`.

Reply:
Pulled this out into a PR: https://github.com/absinthe-graphql/absinthe/pull/743

Reply:
@binaryseed I think that PR looks good. I'm just going to merge it if tests pass.